e2e/test/iothub/device/MethodE2ECustomPayloadTests.cs

```diff
@@ -153,7 +152,8 @@ public async Task Method_DeviceReceivesMethodAndResponse_DictionaryPayload_Amqp(
 
             VerboseTestLogger.WriteLine($"{nameof(ServiceSendMethodAndVerifyResponseAsync)}: Method status: {methodResponse.Status}.");
             methodResponse.Status.Should().Be(200);
-            JsonConvert.SerializeObject(methodResponse.Payload).Should().BeEquivalentTo(JsonConvert.SerializeObject(response));
+            methodResponse.TryGetPayload(out T actual).Should().BeTrue();
+            JsonConvert.SerializeObject(actual).Should().BeEquivalentTo(JsonConvert.SerializeObject(response));
```

Comment:
`JsonConvert.SerializeObject(actual).Should().Be(JsonConvert.SerializeObject(response));`?

String assertions can use `Should().Be(...)`

Reply:
In this case, it is of type T and I don't want to special case strings.